radish34/messenger/src/utils/generalUtils.js

```diff
@@ -14,6 +14,10 @@ const {
 
 const radishApiUrl = process.env.RADISH_API_URL ? `${process.env.RADISH_API_URL}/api/v1` : 'http://localhost:8101/api/v1';
 
+/**
+ * Function that checks whether given object has a JSON structure?
+ * @param {Object} str 
```

Comment:
`str` should be type `String`. This function attempts to convert that string back into an object. There is some extra complexity to catch some edge cases for `str === "true"` and `str === "null"`. Can we change the comment to the following?

```
/**
 * Checks whether a given string can be converted to a proper JSON object.
 * Edge cases: inputs of "true" or "null" return false
 * @param {String} str 
 */
```